information_requests/migrations/0004_complaint_user_complaint.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.8 on 2021-11-21 22:14
```

Comment:
Seria bom adicionarmos apenas uma migration ao invés de duas.

Reply:
@anapaulagomes, eu tava com um probleminha chato de migrations por aqui! 😢  eu coloquei o  `user_complaint` para diferenciar do user do information requests anteriormente porque aparentemente resolvia um erro que tem aparecido aqui na hora que eu vou rodar as migrations: 

django.db.utils.ProgrammingError: column "user_id" of relation "information_requests_informationrequest" already exists

Já fiz o processo de resetar as migrations e já alterei o user_complaint para somente user, e o erro agora é: 
django.db.utils.ProgrammingError: column information_requests_complaint.user_id does not exist

Reply:
Esse erro acontece quando eu tento acessar a página de Denúncias no admin :)

Reply:
vou subir as alterações para ficar mais fácil de você entender!